draft-ietf-mls-protocol.md

```diff
@@ -1477,9 +1490,12 @@ struct {
 } MLSCiphertext;
 ~~~~~
 
+External sender types are sent as MLSPlaintext, see {{external-proposals}}
+for their use.
+
 The remainder of this section describes how to compute the signature of
-an MLSPlaintext object and how to convert it to an MLSCiphertext object.
-The overall process is as follows:
+an MLSPlaintext object and how to convert it to an MLSCiphertext object
+for `member` sender types. The steps are:
```

Comment:
In light of the above discussion, I would suggest adding a step to the encryption where the sender verifies that the `MLSPlaintext.sender.type == member`, then copies `MLSPlaintext.sender.sender` to `MLSCiphertext.sender`.

Reply:
It wasn't quite clear where that'd go; I re-organized the steps to be a little more clear and put the directive for constructing MLSSenderData objects (which is the only place in MLSCiphertext where the sender is identified)